lib/sandbox.js

```diff
@@ -277,6 +277,24 @@ function sandBox(script, name, verbose, debug, context) {
                     + `This warning might become an error in future versions.`
                 );
             }
+            if (!isBinary && (actualCommonType === 'array' || actualCommonType === 'object')) {
+                try {
+                    if (state && isObject(state) && typeof state.val !== 'undefined') {
```

Comment:
```suggestion
                    if (isObject(state) && typeof state.val !== 'undefined') {
```
`isObject` returns `false` for `null` and `undefined`

Reply:
Right